<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExtensionDialog</class>
 <widget class="QDialog" name="ExtensionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>295</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>291</width>
    <height>295</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>291</width>
    <height>295</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Extensions Filter</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>221</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="extension">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addExtension">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add new filter</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="presets">
        <property name="toolTip">
         <string comment="Active preset"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="newPreset">
        <property name="minimumSize">
         <size>
          <width>31</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>31</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string comment="Create new preset"/>
        </property>
        <property name="text">
         <string>New</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removePreset">
        <property name="minimumSize">
         <size>
          <width>51</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>51</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string comment="Remove current preset"/>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="newPresetName">
        <property name="toolTip">
         <string comment="New preset name"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addPreset">
        <property name="maximumSize">
         <size>
          <width>31</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string comment="Create new preset"/>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelPreset">
        <property name="maximumSize">
         <size>
          <width>51</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="SimpleListView" name="presetExtensions">
      <property name="toolTip">
       <string comment="Filters for current preset"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SimpleListView</class>
   <extends>QListView</extends>
   <header>override/simple_list_view.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExtensionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExtensionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
